pubmed_project/settings.py

```diff
@@ -84,32 +84,33 @@
 # ATTENTION !!!
 # While running in local either comment out below section or
 # use 'Production Database Configuration' with Environment Variables.
-# DATABASES = {
-#     'default': {
-#         'ENGINE': 'django.db.backends.postgresql',
-#         'NAME': 'medicles',
-#         'USER': 'postgres',
-#         'PASSWORD': 'postgres',
-#         'HOST': 'db',
-#         'PORT': '5432'
-#     }
-# }
 
-
-# Production Database Configuration with Environment Variables
-# If you use below configuration set local env variables using the values above.
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
-        'NAME': os.environ['DATABASE_NAME'],
```

Comment:
Can we activate this part with environment variables as we're using this in our production?

Reply:
Of course. Sorry, I was sure I changed it. Now it should be good to merge.